labgrid/remote/client.py

```diff
@@ -917,6 +917,8 @@ def digital_io(self):
             drv = target.get_driver("DigitalOutputProtocol", name=name)
         except NoDriverFoundError:
             for resource in target.resources:
+                if not resource.name == name:
```

Comment:
This breaks using `labgrid-client io` without a name. It should be this instead:
```suggestion
                if name and resource.name != name:
```